listenbrainz/db/dump.py

```diff
@@ -46,28 +46,18 @@
 handler.setFormatter(formatter)
 logger.addHandler(handler)
 
-TABLES = {
+# this dict contains the tables dumped in stats dump as keys
+# and a tuple of columns that should be dumped as values
+STATS_TABLES = {
     '"user"': (
         'id',
         'created',
         'musicbrainz_id',
-        'auth_token',
-        'last_login',
-        'latest_import',
-    ),
-    'api_compat.token': (
-        'id',
-        'user_id',
-        'token',
-        'api_key',
-        'ts',
-    ),
-    'api_compat.session': (
-        'id',
-        'user_id',
-        'sid',
-        'api_key',
-        'ts',
+        # the following are dummy values for columns that we do not want to
+        # dump in the stats dump
+        '\'auth_token\'', # auth token
+        'to_timestamp(0)', # last_login
+        'to_timestamp(0)', # latest_import
```

Comment:
I was too tired yesterday when I tried to understand this -- it feels a little odd, but in the general this is a clever way of solving the problem.

Reply:
thanks!